maxim/logger/anthropic/client.py

```
@@ -1,14 +1,14 @@
 from typing import Any
 
-from anthropic import Anthropic
-
 from ..logger import (
     Logger,
 )
+from anthropic import Anthropic
```

Pylint (3.3.7)

[error] Unable to import 'anthropic'

(E0401)